convokit/model/corpus.py

```diff
@@ -44,6 +46,8 @@ def __init__(
         self,
         filename: Optional[str] = None,
         utterances: Optional[List[Utterance]] = None,
+        db_collection_prefix: Optional[str] = None,
```

Comment:
What are your thoughts on putting db arguments in a db_config argument? 

This allows for easier extensibility and maintainability in the long run since we don’t have to change the Corpus constructor if we want to include more configurability for db mode. It also makes the constructor less intimidating for users.

We should have done this for the exclude_ arguments as well IMO, but that’s hindsight for ya